.bazelrc

```diff
@@ -1,12 +1,17 @@
 # Requires at least bazel 6
 common --enable_bzlmod
 
+build --enable_platform_specific_config
+
 # C++, with warnings mostly turned to 11
-build --cxxopt=-xc++                    --host_cxxopt=-xc++
-build --cxxopt=-std=c++23               --host_cxxopt=-std=c++23
-build --cxxopt=-Wall                    --host_cxxopt=-Wall
-build --cxxopt=-Wextra                  --host_cxxopt=-Wextra
-build --cxxopt=-Wno-unused-parameter    --host_cxxopt=-Wno-unused-parameter
+build:macos --macos_minimum_os=10.15
+build:macos --features=-supports_dynamic_linker
+build:macos --cxxopt=-std=c++2b                   --host_cxxopt=-std=c++2b
+build:linux --cxxopt=-std=c++23                   --host_cxxopt=-std=c++23
+build       --cxxopt=-xc++                        --host_cxxopt=-xc++
```

Comment:
Why is there so much additional space here between build and the option ?

Reply:
Ah, I see, because you want to align it with the system-specific ones. ok then (harder to see in the diff, but just looking at the file it became more clear)

Reply:
but the second space is too much. `-Wno-unused-parameter` is the longest option, so that should dictate the table width.